### PR TITLE
Bump up version 4.3.4

### PR DESCRIPTION
**Pull request by myarjunar**
### What does it fix?

* Ticket: #
* Funded by: DFAT
* Description:   
  * Fix bug with map report extent (turns out default template also has extent setting on it)
  * Move the option for using template extent as an Impact Report property

### Checklist:

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Add to the changelog in metadata.txt if it's a new feature
- [ ] Unit test for new code added
- [ ] Request someone to review or test your PR

### CODE REVIEW

File: safe/impact_function/impact_function.py [hunk 1]
```diff
@@ -2898,8 +2899,8 @@ def generate_report(
         :param legend_layers_uri: A list of layers uri for map legend.
         :type legend_layers_uri: list
 
-        :returns: Tuple of error code and message
-        :type: tuple
```
Comment (lucernae) [inline safe/impact_function/impact_function.py]: I believe, at the bottom, we are still returning error codes?

File: safe/report/processors/default.py [hunk 2]
```diff
@@ -431,28 +431,6 @@ def qgis_composer_renderer(impact_report, component):
...
-                extent_element.attributes().namedItem('xmin').nodeValue())
-            min_y = float(
-                extent_element.attributes().namedItem('ymin').nodeValue())
-            max_x = float(
-                extent_element.attributes().namedItem('xmax').nodeValue())
-            max_y = float(
-                extent_element.attributes().namedItem('ymax').nodeValue())
-    except AttributeError:
-        pass  # use default extent
-
```
Comment (lucernae) [inline safe/report/processors/default.py]: Where does this logic goes to?

Comment (lucernae) [reply safe/report/processors/default.py]: I mean, is this going to be deleted, or moved somewhere else?

**Comment by samnawi** [comment]
LGTM

**Comment by felix-yew** [comment]
Hi mas @myarjunar for the multiexposure analysis result, the map result has a map view 👍 but some input for :
1. the landscape map result - the number in legend was cut
image
2. the infographic result, the title text still cut and the result number still empty
image

**Pull request merged**